src/cryptography/x509.py

```diff
@@ -29,6 +32,92 @@ def __init__(self, msg, parsed_version):
         self.parsed_version = parsed_version
 
 
+class UnknownAttribute(Exception):
+    pass
+
+
+Attribute = collections.namedtuple("Attribute", ["oid", "name", "value"])
```

Comment:
Let's use a "normal" class instead of a namedtuple; I dislike them for public APIs because it's a very leak abstraction; e.g. `Attribute("a", "b", "c") == ("a", "b", "c")` or any other namedtuple type.